pkgs/development/libraries/libpointmatcher/default.nix

```diff
@@ -0,0 +1,41 @@
+{stdenv, fetchFromGitHub, cmake, eigen, boost, libnabo}:
+
+let inherit (stdenv.lib) getVersion versionAtLeast; in
+
+assert versionAtLeast (getVersion eigen) "3";
+assert versionAtLeast (getVersion boost) "1.48";
+assert versionAtLeast (getVersion eigen) "1.0.6";
+
+stdenv.mkDerivation rec {
+  version = "7504481";
```

Comment:
I just realized that this is the prefix of the revision and not an actual version. The new guidelines state that packages derived from vcs checkouts shall use the date of the commit in year-month-day format as the version. This is to ensure that version strings satisfy `new > old`.